### ISSUE TITLE
Analytics: Add site details to editor media events

### ISSUE THREAD

**Issue by aforcier** [wordpress-mobile/WordPress-Android#5691]
It looks like we stopped attaching the site id to media events from the editor (`EDITOR_ADDED_PHOTO_NEW`, for example) [about a year ago](https://github.com/wordpress-mobile/WordPress-Android/commit/e11b0acd24055f63a15cebf8cecbe6cbf4edebd1). I don't think it was intentional (unless you remember something I don't @maxme?).

We should probably go back to using `trackWithSiteDetails()` for those events so we can use the `blogid` property in analytics.

Events in question: 
```java
EDITOR_ADDED_PHOTO_VIA_DEVICE_LIBRARY
EDITOR_ADDED_VIDEO_VIA_DEVICE_LIBRARY
EDITOR_ADDED_PHOTO_NEW
EDITOR_ADDED_VIDEO_NEW
EDITOR_ADDED_PHOTO_VIA_WP_MEDIA_LIBRARY
EDITOR_ADDED_VIDEO_VIA_WP_MEDIA_LIBRARY
```

**Comment by maxme** [comment]
Probably not intentional, I remember removing the number of media to simplify the code (and because we never used that).

**Comment by daniloercoli** [comment]
Fixed in #5770
